src/GovUk.Education.ExploreEducationStatistics.Content.Services.Tests/ThemeServiceTests.cs

```diff
@@ -2086,73 +2596,58 @@ await using (var context = InMemoryContentDbContext(contextId))
         }
 
         [Fact]
-        public async Task GetPublicationTree_LatestData_FiltersThemesWithNoVisiblePublications()
+        public async Task GetPublicationTree_DataCatalogue_DoesNotFilterPublicationWithDataOnLatestRelease()
```

Comment:
This probably makes more sense as `DoesNotFilterPublicationWithoutDataOnLatestRelease`

Reply:
I've gone with `DoesFilterPublicationWithoutDataOnLatestRelease` - I imagine this is what you intended.